edx_django_utils/monitoring/internal/backends.py

```diff
@@ -51,7 +51,7 @@ def record_exception(self):
         """
 
     @abstractmethod
-    def create_span(self, name):
+    def create_span(self, name, **kwargs):
```

Comment:
I was imaging that the API would be well defined, and not completely open. Note that someone using this API needs to write a single line of open source code (e.g. `create_span(name, xxx)` etc.) that should do the right thing no matter what backend is configured.  So, I imagining a well defined second argument, with documentation about how it is treated for each implementing backend.